oauth/authentication-oauth2.0.md

```diff
@@ -9,26 +9,26 @@ SolveXia API provides two main methods for generating access tokens: **Client Cr
 
 We strongly recommend using Authorise Code Flow for client production applications. 
 The Client Credentials Flow is intended to be used by internal scripts or for testing purposes. 
-Public third party applications that rely on GitHub for authentication should not ask for or collect SolveXia user credentials.
+Public third party applications that rely on SolveXia for authentication should not ask for or collect SolveXia user credentials.
 Instead, they must use the Authorise Code Flow web flow.
 
-### Table of Contents
+
+### Index
 [Client Credentials Flow](#client-credentials-flow)  
-[Authorization Code Flow](#authorize-code-flow)  
+[Authorization Code Flow](#authorize-code-flow)
+[Refresh Token](#refresh-token)  
 [API Request with Token](#call-solveXia-public-api-with-access-token)
 
----
 ## Client Credentials Flow
 
-Client Credentials Flow is available for limited contexts like internal scripts or testing.
-
-The flow contains the following steps:
-1. Generate access token by supplying client_id and client_secret
+Client Credentials Flow is available for limited contexts like internal scripts or testing. 
+The flow contains just one step which generates access token by supplying client_id and client_secret.
```

Comment:
Should be "generates an access token" (if singular) or generates access tokens" (if plural)

Reply:
Thanks, fixed.